modules/test/playwright/tests/notification-web/emailNotificationTemplate.spec.ts

```diff
@@ -154,10 +162,26 @@ test('can save recipients roles in email notification template', async ({
 
 	await emailNotificationTemplatePage.contentSubject.fill('Content subject');
 
+	await page.getByRole('button').filter({hasText: 'Copy'}).first().click();
+
+	await emailNotificationTemplatePage.CKEditor.click();
+
+	await page.keyboard.press('Control+V');
+
+	await emailNotificationTemplatePage.entity.click();
+
+	await page.getByRole('option', {name: 'Postal Address'}).click();
+
+	await page.getByRole('button').filter({hasText: 'Copy'}).last().click();
+
+	await emailNotificationTemplatePage.CKEditor.click();
+
+	await page.keyboard.press('Control+V');
```

Comment:
Hey @igor-franca, I think it's good to keep testing the copy functionality here, but if we really want complete coverage of this functionality, what do you think about create a unit test for `copyGeneralTerm` function from `GeneralTerms.tsx`?